src/main/java/seedu/manager/ui/Ui.java

```diff
@@ -34,16 +30,16 @@ public void greetUser() {
     /**
      * Gets input from the user, and executes commands based on that input
      */
-    public void getCommands() {
-        boolean isGettingCommands = true;
-
-        while (isGettingCommands) {
-            System.out.print(COMMAND_PROMPT_MESSAGE);
-            String rawInput = this.userInput.nextLine();
-            Command command = parser.parseCommand(rawInput);
-            CommandOutput output = command.execute();
-            output.printMessage();
-            isGettingCommands = !output.getCanExit();
-        }
+    public String getCommands() {
```

Comment:
I think you could change the method name to getCommand, since the method only parses for one command at a time?